tests/test_macos.py

```diff
@@ -94,6 +96,10 @@ def test_macos_homebrew(mocker: MockerFixture, params: dict[str, Any], multipath
     suffix_elements = tuple(params[i] for i in ("appname", "version") if i in params)
     suffix = os.sep.join(("", *suffix_elements)) if suffix_elements else ""  # noqa: PTH118
 
+    expected_path_map = {
+        "site_cache_path": Path(f"/opt/homebrew/var/cache{suffix}"),
+        "site_data_path": Path(f"/opt/homebrew/share{suffix}"),
+    }
     expected_map = {
```

Comment:
I split the paths up because `mypy` isn't happy with the addition in the `if multipath` block when the values can be `str | Path`